tests/utils/mock_server.py

```diff
@@ -410,6 +412,10 @@ def graphql():
         if body["variables"].get("files"):
             requested_file = body["variables"]["files"][0]
             ctx["requested_file"] = requested_file
+            emulate_azure = ctx.get("emulate_azure")
+            # Azure expects the request path of signed urls to have 2 parts
+            if emulate_azure:
+                base_url += "/storage"
```

Comment:
So `url` will have `/storage/storage`?

Reply:
Exactly, the azure library assumes all storage urls have at least 2 path components